### PR TITLE
feat(docs): FRMW-202 - Migrate rxToggleSwitch docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-201

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxToggleSwitch/rxToggleSwitch.js [hunk 1]
```diff
@@ -15,8 +15,23 @@ angular.module('encore.ui.rxToggleSwitch', [])
...
+ * `true-value` and `false-value` attributes.
+ *
+ * ** Note: If the value of the model is not defined at the time of
+ * initialization, it will be automatically set to the false value. **
+ *
+ * The expression passed to the `post-hook` attribute will be evaluated every
+ * time the switch is toggled (after the model property is written on the
+ * scope).  It takes one argument, `value`, which is the new value of the model.
+ * This can be used instead of a `$scope.$watch` on the `ng-model` property.
+ * As shown in the demo, the `disabled` attribute can be used to prevent further
```
Comment (100stacks) [inline src/rxToggleSwitch/rxToggleSwitch.js]: Suggest adding a demo link:
`{@link /#/components/rxToggleSwitch demo}`

Comment (bry4n) [inline src/rxToggleSwitch/rxToggleSwitch.js]: :+1:

**Comment by 100stacks** [comment]
LGTM :ok_hand:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**